packages/app-layout/src/AppLayout.stories.tsx

```diff
@@ -265,3 +265,55 @@ export const BusinessDropdownModalInterrupt: Story = {
 		);
 	},
 };
+
+export const Claims: Story = {
+	args: {
+		focusMode: false,
+		unreadMessageCount: 6,
+		activePath: '/',
+		claims: {
+			given_name: 'given_name',
+			family_name: 'family_name',
+		},
+	},
+};
+
+export const ClaimsMissingName: Story = {
+	args: {
+		focusMode: false,
+		unreadMessageCount: 6,
+		activePath: '/',
+		claims: {
+			family_name: 'family_name',
+		},
+	},
+};
+
+export const ClaimsMissingNameCustom: Story = {
+	args: {
+		focusMode: false,
+		unreadMessageCount: 6,
+		activePath: '/',
+		claims: {
+			family_name: 'family_name',
+		},
+		errorComponents: {
+			MissingGivenName: (props: PropsWithChildren) => {
```

Comment:
I wonder if we should have a catch-all error component (`MissingName`), because I can imagine teams just want to render the same content (the app) regardless which name was missing. That also aligns with your general MissingName variable below

Reply:
agree in principle but account team has implied to me that the two situations have very different semantics so I am hesitant to merge them